docs/json_transformations.md

```diff
@@ -16,56 +16,53 @@ can boil down the burden of implementing a custom serializer to a problem of man
 
 It is still strongly recommended to be acquainted with [custom serializers guide](custom_serializers.md)
 in its [using serializers](custom_serializers.md#using-custom-serializers) part,
-since the instantiation process and annotations to enable custom serializers remain the same.
-It is also important to remember that since the discussed in this article serializers are manipulating Json AST,
-they have the following limitations:
+since the instantiation process and annotations to enable custom serializers remains the same.
```

Comment:
Isn't `remain` correct because it refers to plural ' instantiation process and annotations'?